alpine-rails-nginx/Dockerfile

```diff
@@ -4,8 +4,8 @@ FROM alpine:edge
 MAINTAINER Dan Kubb <dkubb@fastmail.com>
 
 # Create system users
-RUN adduser -DS nginx
-RUN adduser -DS rails
+RUN adduser -DSH -G nogroup nginx \
```

Comment:
There are no short options on Alpine Linux's `adduser`. Many of it's binaries are paired down and are not fully compatible with the common gnu alternatives.